### DISCUSSION TITLE
How to know if the issue has been considered/triaged?

### DISCUSSION THREAD

**Discussion by andrii-korotkov-verkada** [argoproj/argo-cd#18167] [Q&A] [answered]
Hey. I hope your day is going well. I've raised a couple of small enhancements ([link1](https://github.com/argoproj/argo-cd/issues/18113), [link2](https://github.com/argoproj/argo-cd/issues/18114)) and consider even implementing myself if contributors don't have enough time. The [contributions guide](https://argo-cd.readthedocs.io/en/latest/developer-guide/code-contributions/) says that issues need to be triaged and accepted before the contributions can be accepted. However, the [linked project](https://github.com/orgs/argoproj/projects/18) has not been updated for ~1.5 years. Seeing a sheer number of open issues, I understand that contributors are quite busy, but still wonder when the issues might have a chance to go through the triage and be accepted. Do you know of the current expected wait time for enhancements? Thank you.

**Comment by crenshaw-dev** [comment, marked as answer]
It depends heavily on the PR. For your proposed enhancements (additional logging) I'd expect pretty fast reviews. 

You can get a general idea about the lead time of different-sized PRs here: https://insights.lfx.linuxfoundation.org/foundation/cncf/velocity?project=argo&routedFrom=Github&bestPractice=false&repository=https:%2F%2Fgithub.com%2Fargoproj%2Fargo-cd

**Reply by andrii-korotkov-verkada** [reply]
Thanks! In general, for small enhancements and bug fixes, is it okay to just submit PR(s)?

**Reply by crenshaw-dev** [reply]
Yep, that works!
